### ISSUE TITLE
The executable 'bsdtar' Vagrant is trying to run was not found in the %PATH% variable

### ISSUE THREAD

**Issue by dcarrith** [hashicorp/vagrant#3674]
I just upgraded to Vagrant v.1.6.0 and now I'm getting this error when trying to add a box:

``` shell
PS C:\Vagrant\packer-example-master> vagrant box add --name trusty-tahr-0.1b "C:\Vagrant\packer-example-master\boxes\virtualbox\ubuntu1404-provisionerless0.1b.box"
==> box: Adding box 'trusty-tahr-0.1b' (v0) for provider:
    box: Downloading: file://C:/Vagrant/packer-example-master/boxes/virtualbox/ubuntu1404-provisi
onerless0.1b.box
    box: Progress: 100% (Rate: 53.9M/s, Estimated time remaining: --:--:--)
The executable 'bsdtar' Vagrant is trying to run was not
found in the %PATH% variable. This is an error. Please verify
this software is installed and on the path.
```

**Comment by jpmurray** [comment]
Encountered the same problem, as @djberg96 stated, repairing the install worked perfectly.

**Comment by tatarinov** [comment]
I fix problem "The box failed to unpackage properly. Please verify that the box
file you're trying to add is not corrupted and try again. The
output from attempting to unpackage (if any):" in Windows XP 32.

Rename "C:\HashiCorp\Vagrant\embedded\gnuwin32\bin" in "C:\HashiCorp\Vagrant\embedded\gnuwin32\bin.old"
Create folder "C:\HashiCorp\Vagrant\embedded\gnuwin32\bin"
Download:
http://downloads.sourceforge.net/gnuwin32/libarchive-2.4.12-1-bin.zip
http://gnuwin32.sourceforge.net/downlinks/bzip2-bin-zip.php
http://gnuwin32.sourceforge.net/downlinks/zlib-bin-zip.php
Unpacked all. Copy files from all bin/*.exe and bin/*.dll in to "C:\HashiCorp\Vagrant\embedded\gnuwin32\bin".

**Comment by iakio** [comment]
Anyway, It would be better to check bsdtar.exe befor downloading.

**Comment by lms-it** [comment]
I was unable to run "bsdtar --version" ... adding the zlib1.dll to \gnuwin32\bin directory as @tatarinov mentioned fixed the problem.

**Comment by johnklehm** [comment]
I'm still seeing this issue with vagrant 1.7.2 on Windows 7 with a limited rights user account.  Manually adding the C:\HashiCorp\Vagrant\embedded\gnuwin32\bin folder to my path as admin resolved the issue.  As to why I'd have this setup...corporate rules.  Is it possible for vagrant to set the path for bsdtar in a way that doesn't require admin escalation other than at install time perhaps an additional environment variable?

**Comment by oneminot** [comment]
On Windows 8.1. It seems I only get this message when I try to vagrant up from git bash. It did not happen for me on cmd. If you get this message in WIndows, please try running `vagrant up` from a cmd window.

**Comment by marcamon2013** [comment]
Just add C:\HashiCorp\Vagrant\embedded\mingw\bin in your PATH solved this issue on Windows 10 with vagrant 1.7.1

**Comment by bumblebee99** [comment]
Faced this problem with git bash. Used windows cmd for vagrant box add and it works properly.
